config/license/allowlist.json

```diff
@@ -9,6 +9,24 @@
       "actualLicense": "MIT",
       "violationAllowance": ""
     },
+    {
+      "moduleLicense": "Apache-2.0",
+      "moduleVersion": "6.4.0",
+      "moduleName": "importlib-resources",
+      "licenseUrl": "https://github.com/python/importlib_resources/blob/v6.4.0/LICENSE",
+      "moduleCopyright": "Barry Warsaw and the contributors to the project",
```

Comment:
i guess the [copyrights ](https://github.com/jaraco/skeleton/issues/78) were deleted within the source files?